stdpopsim/genomes.py

```diff
@@ -49,6 +50,7 @@ def from_data(genome_data, *, recombination_rate, mutation_rate, citations):
                     recombination_rate=recombination_rate[name],
                 )
             )
+        append_common_synonyms(chromosomes)
```

Comment:
I don't think we should do it here - best to make a case-by-base choice when adding the species.

Reply:
Ok, makes sense. I'll reformat a bit.

Reply:
I moved it around a bit to append synonyms to a genome object, so that it doesn't affect the current default setup of the species.py file. Can then choose to either add or not to that file after calling `_genome = species.Genome.from_data(...)`.